backend/api/views.py

```diff
@@ -357,9 +357,9 @@ def verify(self, request, year):
                 "message": "You already have finished your payment."
             })        
 
-        if request.GET.get('Status') == 'OK':
+        if request.query_params.get('Status', None) == 'OK':
             amount = wss.registration_fee
-            result = verify(request.GET['Authority'], amount)
+            result = verify(request.query_params.get('Authority', None), amount)
```

Comment:
What happens if it's `None`? Isn't it better to hard-check and raise exception?

Reply:
Done!